app/src/main/java/com/cornellappdev/transit/ui/screens/RouteScreen.kt

```diff
@@ -0,0 +1,164 @@
+                Text(
+                    text = "From",
+                    color = PrimaryText,
+                    fontFamily = sfProDisplayFamily,
+                    fontWeight = FontWeight.Normal
+                )
+                Text(
+                    text = "To",
+                    color = PrimaryText,
+                    fontFamily = sfProDisplayFamily,
+                    fontWeight = FontWeight.Normal
+                )
+            }
+
+            Icon(
+                painter = painterResource(id = R.drawable.route_icon),
+                contentDescription = "",
+                modifier = Modifier.size(width = 16.dp, height = 54.dp),
+                tint = Color.Unspecified
+            )
+
+            //TODO user should be able to enter location/destination
+            Column(verticalArrangement = Arrangement.spacedBy(12.dp)) {
+                Box(
+                    modifier = Modifier
+                        .background(color = DividerGrey, shape = RoundedCornerShape(8.dp))
+                        .fillMaxWidth(0.9f)
+                ) {
+                    Text(
+                        text = "Current Location",
+                        modifier = Modifier.padding(horizontal = 12.dp, vertical = 6.dp),
+                        color = PrimaryText,
+                        fontFamily = sfProDisplayFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+                Box(
+                    modifier = Modifier
+                        .background(color = DividerGrey, shape = RoundedCornerShape(8.dp))
+                        .fillMaxWidth(0.9f)
+                ) {
+                    Text(
+                        text = "Destination",
+                        modifier = Modifier.padding(horizontal = 12.dp, vertical = 6.dp),
+                        fontFamily = sfProDisplayFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+            }
+
+            //TODO this should probably be an IconButton that swaps current/dest
+            Icon(
+                painter = painterResource(id = R.drawable.swap_icon),
+                contentDescription = "",
+                modifier = Modifier.size(width = 20.dp, height = 20.dp),
+                tint = Color.Unspecified
+            )
+
+        }
+
+        Divider(thickness = 1.dp, color = DividerGrey)
+
+        //TODO the text should be set to what the time is (passed in by screen call?)
+        Row(modifier = Modifier.padding(horizontal=20.dp,vertical=12.dp), verticalAlignment = Alignment.CenterVertically, horizontalArrangement = Arrangement.spacedBy(12.dp)) {
+            Icon(
+                painter = painterResource(id = R.drawable.clock_icon),
+                contentDescription = "",
+                modifier = Modifier.size(width = 12.dp, height = 12.dp),
+                tint = Color.Unspecified
+            )
```

Comment:
Make this an SVG too! Also, I could be crazy but it looks to be slightly vertically misaligned, which is weird since you have center vertically there... maybe mess around with just adding a tiny padding if this persists after SVG.